tools/BuildAssets/targets/common.Build.props

```diff
@@ -18,38 +18,29 @@
 		<BuildWideDebugTrace Condition=" '$(DebugTrace)' == '' ">false</BuildWideDebugTrace>
```

Comment:
You probably need to condition DelaySign property above to make sure it doesn't override something setting it before like the buildtools.props.

Reply:
@weshaggard you mean, check for if it's set to false and if yes, set it to true? Is that what you mean by conditioning the setter for the DelaySign?
Also buildtools.props is not included in any build workflow, unless you are building the buildtools projects. Do you still anticipate buildtools.prop overriding any defaults for normal workflow?
Also buildtools.props will never be merged to the master repo.

Reply:
I was meaning something like `<DelaySign Condition="'$(DelaySign)' == ''>true</DelaySign>` . Just default it to true if it isn't already set to something else.

Reply:
@weshaggard but dont' we want to just set it to true by default regardless of what the existing value is?
And it will get overridden while building build tools project to false.
Maybe I am missing to understand why it has to be conditioned in the global scope?

Reply:
Assuming nothing imported before this is expected to set DelaySign then it is fine to always set it explicitly.